protected/migrations/m170823_064156_cera_ethnic_groups.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+class m170823_064156_cera_ethnic_groups extends OEMigration
+{
+	public function safeUp()
+	{
+        /**
```

Comment:
Formatting is a bit funny, but being a migration, it doesn't matter all that much